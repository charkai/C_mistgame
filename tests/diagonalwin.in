place I15
place J15
place J16
place J17
place K17
place A2
place L18
place B1
place M19
