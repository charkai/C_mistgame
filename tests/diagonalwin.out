Black wins!
I15J15J16J17K17A2L18B1M19
Thank you for playing!
